tests/Doctrine/Tests/DBAL/Functional/Driver/PDOSqlsrv/DriverTest.php

```diff
@@ -64,11 +63,4 @@ public function testConnectionOptions() : void
 
         self::assertSame('APP_NAME', $result);
     }
-
-    public function testDriverOptions() : void
```

Comment:
Is this test breaking with your changes?

Reply:
Yes, because previously Connection extended PDOConnection, now it only implements Connection interface, where is no method like getAttribute.

Reply:
master-branch is currently for 2.10 while develop is for 3.0. It looks like the changes are more fitting for the next major release.

Reply:
So should I make a branch based on develop then apply these changes and create a new pull request or just override this?